lib/bom.js

```diff
@@ -812,8 +812,8 @@ declare class Headers {
     append(name: string, value: string): void;
     delete(name: string): void;
     entries(): Iterator<[string, string]>;
+    forEach((value: string, name: string, index: number) => any): void;
```

Comment:
Good catch; thanks! I've also noticed that I'm missing the `thisArg` optional argument.